docs/python-api/introduction.rst

```diff
@@ -1,35 +1,35 @@
-Introduction
-============
-|neptune-client-github| is open source Python library that allows you to integrate your Python scripts with Neptune. Neptune client supports handful of use cases:
+About neptune-client
+--------------------
 
-* creating and tracking experiments
-* managing running experiment
-* querying experiments and projects (search/download)
+|neptune-client-github|
+is an open source Python library that lets you integrate your Python scripts with Neptune so that you can more easily track and organize your experiments in the rich Neptune `dashboard <https://ui.neptune.ai/shared/onboarding/experiments>`_.
 
-.. note:: Make sure to register to |neptune|, to use it.
+Once you have integrated with Neptune, you can also:
 
-Neptune implements client-server architecture. Because of that you can log and access your results from many different devices:
+* Create experiments. `Example <https://ui.neptune.ai/USERNAME/example-project/e/HELLO-48/source-code?path=.&file=classification-example.py>`_.
+* Manage running experiments. `Example <https://ui.neptune.ai/USERNAME/example-project/e/HELLO-48/source-code?path=.&file=classification-example.py>`_.
+* Query experiments and projects. `Example <https://ui.neptune.ai/USERNAME/example-project/n/Experiments-analysis-with-Query-API-and-Seaborn-31510158-04e2-47a5-a823-1cd97a0d8fcd/91350522-2b98-482d-bc14-a6ff5c061b6b>`_.
 
-* laptops
-* cluster of machines
-* cloud services
+The `Neptune Python Library reference <api-reference.html>`_ provides a complete description of these capabilities.
 
-.. image:: ../_static/images/python_api/server_client_arch.png
-   :target: ../_static/images/python_api/server_client_arch.png
-   :alt: basic architecture
+.. note:: You must register with |neptune| to use neptune-client.
 
 .. _installation:
 
 Installation
-------------
+============
+
 .. code:: bash
 
     pip install neptune-client
 
-Once installed, ``import neptune`` in your code to use it.
+Once installed, add ``import neptune`` to your code to use neptune-client capabilities.
 
 Example
--------
+=======
```

Comment:
Here is better example code, only slightly longer but presents a lot more value:


```
import neptune
import numpy as np

# select project
neptune.init('USERNAME/example-project',
             api_token='ANONYMOUS')

# define parameters
PARAMS = {'decay_factor': 0.7,
          'n_iterations': 117}

# create experiment
neptune.create_experiment(name='quick_start_example',
                          params=PARAMS)

# log some metrics
for i in range(1, PARAMS['n_iterations']):
    neptune.log_metric('iteration', i)
    neptune.log_metric('loss', PARAMS['decay_factor']/i**0.5)
    neptune.log_text('text_info', 'some value {}'.format(0.95*i**2))

# add tag to the experiment
neptune.append_tag('quick_start')

# log some images
for j in range(5):
    array = np.random.rand(10, 10, 3)*255
    array = np.repeat(array, 30, 0)
    array = np.repeat(array, 30, 1)
    neptune.log_image('mosaics', array)
```